src/pages/search/advanced_search/LanguageFilter.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Chip, FormControl, InputLabel, MenuItem, Select } from '@mui/material';
+import { Language } from 'nva-language';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { ResultParam } from '../../../api/searchApi';
+import i18n from '../../../translations/i18n';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { languageOptions } from '../../registration/DescriptionPanel';
+
+export const LanguageFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const languageParam = searchParams.get(ResultParam.PublicationLanguageShould)?.split(',') || [];
+  const [open, setOpen] = useState(false);
+
+  const toggleOpenOptions = () => {
+    setOpen(!open);
+  };
+
+  const updateSelectedLanguages = (selectedUris: string[] | null) => {
```

Comment:
Setter typen til `as string[]` hvor denne kalles
```suggestion
  const updateSelectedLanguages = (selectedUris: string[]) => {
```